assets/schema_input.json

```
@@ -181,6 +172,17 @@
                 "type": "string"
             }
         },
+        "uniqueEntries": ["patient", "sample"],
+        "dependentRequired":{
+            "anyOf": [
+                {
+                    "lane": ["fastq_1"]
+                },
+                {
+                    "lane": ["bam"]
+                }
+            ]
+        },
         "required": ["patient", "sample"]
     }
 }
```

```suggestion
        "dependentRequired":{
            "anyOf": [
                {
                    "lane": ["fastq_1"]
                },
                {
                    "lane": ["bam"]
                }
            ]
        },
        "required": ["patient", "sample"]
    },
    "uniqueEntries": ["patient", "sample"]
}
```
This should do the trick :p